codebundles/aws-c7n-ec2-health/.runwhen/templates/aws-c7n-ec2-health-taskset.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: runwhen.com/v1
+kind: Runbook
+metadata:
+  name: {{slx_name}}
+  labels:
+    {% include "common-labels.yaml" %}
+  annotations:
+    {% include "common-annotations.yaml" %}
+spec:
+  location: {{default_location}}
+  description: Runs tasks to identify and triage unused and unpatched AWS EC2 Instances.
```

Comment:
Same comment here. Remove the word triage. Remove the word unused. Switch unused to "stale"

e.g. "Identify stale and unpatched AWS EC2 Instances in AWS Region ${REGION} for AWS Account ${ACCOUNT}"